lib/hex_web/endpoint.ex

```diff
@@ -30,5 +30,12 @@ defmodule HexWeb.Endpoint do
   plug Plug.MethodOverride
   plug Plug.Head
 
+  plug HexWeb.Session,
+    store: :cookie,
+    key: "_hex_web_key",
+    signing_salt: "vGpjq82L",
+    encryption_salt: "y77stVMu",
```

Comment:
Better to move them to ENV. You are right those are used to derive keys on top of the Phoenix secret but I wouldn't risk to expose them.